src/zope/interface/tests/test_interface.py

```diff
@@ -2116,17 +2117,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
 
 def test_suite():
-    import doctest
     return unittest.TestSuite((
-        unittest.makeSuite(ElementTests),
-        unittest.makeSuite(SpecificationBasePyTests),
-        unittest.makeSuite(InterfaceBasePyTests),
-        unittest.makeSuite(SpecificationTests),
-        unittest.makeSuite(InterfaceTests),
-        unittest.makeSuite(AttributeTests),
-        unittest.makeSuite(MethodTests),
-        unittest.makeSuite(Test_fromFunction),
-        #unittest.makeSuite(Test_fromMethod),
-        doctest.DocTestSuite(),
+        unittest.defaultTestLoader.loadTestsFromName(__name__),
         doctest.DocTestSuite("zope.interface.interface"),
     ))
```

Comment:
There are some that feel that testing code snippets in documentation is important. I find that a sympathetic argument (more so than other arguments for doctests). However, none of the docstrings contained in zope.interface.interface actually have any doctest snippets. A module that it imports *does*, but doctest doesn't run that. 

So this seems to serve no point at all.